### ISSUE TITLE
qemu_arc_{em,hs} keeps failing in CI on tests/kernel/lifo/lifo_usage

### ISSUE THREAD

**Issue by nashif** [zephyrproject-rtos/zephyr#26163]
**Describe the bug**
Keeps failing in CI

https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/75382/3/tests

```
*** Booting Zephyr OS build zephyr-v2.3.0-305-gb04ff83884de  ***
Running test suite test_lifo_usage
===================================================================
starting test - test_lifo_nowait
PASS - test_lifo_nowait
===================================================================
starting test - test_lifo_wait
PASS - test_lifo_wait
===================================================================
starting test - test_timeout_empty_lifo
PASS - test_timeout_empty_lifo
===================================================================
starting test - test_timeout_non_empty_lifo
PASS - test_timeout_non_empty_lifo
===================================================================
starting test - test_timeout_lifo_thread

    Assertion failed at WEST_TOPDIR/zephyr/tests/kernel/lifo/lifo_usage/src/main.c:329: test_timeout_lifo_thread: packet != NULL is false

FAIL - test_timeout_lifo_thread
===================================================================
starting test - test_timeout_threads_pend_on_lifo

    Assertion failed at WEST_TOPDIR/zephyr/tests/kernel/lifo/lifo_usage/src/main.c:401: test_thread_pend_and_timeout: packet == NULL is false

FAIL - test_timeout_threads_pend_on_lifo
===================================================================
Test suite test_lifo_usage failed.
===================================================================
PROJECT EXECUTION FAILED
```

excluding it for now, but need a fix.

EDIT: This test fails on both `qemu_arc_hs` and `qemu_arc_em`.

**Comment by stephanosio** [comment]
This test fails on both `qemu_arc_hs` and `qemu_arc_em`.

**Comment by nashif** [comment]
closed by #31550
